Microsoft.Toolkit.Uwp.SampleApp/SamplePages/WrapLayout/WrapLayout.bind

```diff
@@ -19,14 +19,18 @@
 
   <Grid Padding="48">
     <winui:ItemsRepeaterScrollHost> <!-- Needed for 1803 and below -->
-      <ScrollViewer>
+      <ScrollViewer 
```

Comment:
We may just want to take the page out of the similar problem in the `WrapPanel` sample and add a button to switch the orientation:

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/9b15f6ab571ead581a969dba471e73807f4fa175/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/WrapPanel/WrapPanelPage.xaml.cs#L47

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/9b15f6ab571ead581a969dba471e73807f4fa175/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/WrapPanel/WrapPanelPage.xaml.cs#L83-L104

This then takes out any guesswork to proper combination from the developer looking at the sample

Reply:
thx Michael. I'll apply this.